cypress/e2e/error-handling.cy.ts

```diff
@@ -0,0 +1,71 @@
+describe("Server-side error handling", () => {
```

Comment:
I wasn't able to pull this down and actually run the tests in Cypress but at a glance this looks good!